web/regression/re_sql/tests/test_resql.py

```
@@ -848,11 +848,13 @@ def preprocess_expected_sql(self, scenario, sql, resp_sql, object_id):
 
             # Fetch the lc_collate and lc_ctype
             pg_cursor.execute(
-                "SELECT datcollate as cname FROM pg_database WHERE datname = "
+                "SELECT datcollate as cname FROM pg_catalog.pg_database "
+                "WHERE datname = "
                 "'{0}'".format(db_name))
```

Ruff (0.15.15)

[error] Possible SQL injection vector through string-based query construction

(S608)